feature/google/src/test/java/BillingWrapperCommonTest.kt

```diff
@@ -482,29 +358,39 @@ class BillingWrapperTest {
     @Test
     fun queryHistoryCallsListenerIfOk() {
         billingClientStateListener!!.onBillingSetupFinished(billingClientOKResult)
+
+        mockClient.mockQueryPurchaseHistory(
+            billingClientOKResult,
+            emptyList()
+        )
+
         var successCalled = false
+        val type = ProductType.SUBS.toGoogleProductType()
         wrapper.queryPurchaseHistoryAsync(
-            BillingClient.SkuType.SUBS,
+            type!!,
             {
                 successCalled = true
             },
             {
                 fail("shouldn't go to on error")
             }
         )
-        billingClientPurchaseHistoryListener!!.onPurchaseHistoryResponse(
-            billingClientOKResult,
-            ArrayList()
-        )
-        assertThat(successCalled).isTrue()
+        assertThat(successCalled).isTrue
     }
 
     @Test
-    fun queryHistoryNotCalledIfNotOK() {
+    fun queryHistoryErrorCalledIfNotOK() {
         billingClientStateListener!!.onBillingSetupFinished(billingClientOKResult)
+
+        mockClient.mockQueryPurchaseHistory(
+            BillingClient.BillingResponseCode.FEATURE_NOT_SUPPORTED.buildResult(),
+            emptyList()
+        )
+
         var errorCalled = false
+        val type = ProductType.SUBS.toGoogleProductType()
```

Comment:
Should we extract this to a `@Before` block and store it in a variable like `subsGoogleProductType`? That way we don't need to add the `!!` on each test. We can have it for both `SUBS`. and `INAPP`. Also this could be moved to the base test class.